src/components/Post.jsx

```diff
@@ -20,7 +61,7 @@ class Post extends React.Component{
               Written at: {this.props.timestamp}
             </div>
             <button> Edit </button>
-            <button> Delete </button>
+            <button onClick={this.handleClick}> Delete </button>
```

Comment:
might have to rename handleClick to something specific later when Edit button is set up